docs/Reply.md

```diff
@@ -8,6 +8,7 @@ Reply is a core Fastify object that exposes the following functions:
 - `.header(name, value)` - Sets the headers.
 - `.type(value)` - Sets the header `Content-Type`.
 - `.redirect([code,] url)` - Redirect to the specified url, the status code is optional (default to `302`).
+- `.serialize(reply, payload)` - The default json serializer.
```

Comment:
It's not entirely clear to me why this would need both parameters. The documentation should make it clear to me.